endpoints.json

```diff
@@ -53,5 +53,19 @@
       " body": "Delicious crackerbreads",
       " article_id": 1
     }
+  },
+  "POST /api/articles/:article_id/comments": {
+    "description": "posts a new comment to the comments table with the id of the parameter article",
+    "parameter": ["number of article"],
```

Comment:
Is this referencing the parametric endpoint? If so then this doesnt need to be here as you already have this visible on line 57 with the `:article_id` syntax.